testsuite/tests/ImportExportUser.out

```diff
@@ -2,6 +2,6 @@ Dump	==========================================================
 Dump	importing:
 Dump	 $["encrypted":true, "forename":"aa", "gid":500, "grouplist":"audio", "home":"/home/aaa", "password_settings":$["inact":-1, "max":99999, "min":0, "warn":7], "shell":"/bin/bash", "surname":"A", "uid":500, "user_password":"5OkhN2nLxMfEY", "username":"aaa"]
 Dump	==== processing ImportUser... ============================
-Return	$["authorized_keys":[], "cn":"aa A", "encrypted":true, "gidNumber":"500", "grouplist":$["audio":"1"], "homeDirectory":"/home/aaa", "loginShell":"/bin/bash", "modified":"imported", "shadowInactive":"-1", "shadowLastChange":"0", "shadowMax":"99999", "shadowMin":"0", "shadowWarning":"7", "type":"local", "uid":"aaa", "uidNumber":"500", "userPassword":"5OkhN2nLxMfEY"]
```

Comment:
AFAIS, we import `btrfs_subvolume` and export `home_btrfs_subvolume`?

Reply:
Yes, I know "btrfs_subvolume" is the internal variable. I have used "home_btrfs_subvolume" for the AutoYaST file. It is more userfriendly.